.devcontainer/devcontainer.json

```diff
@@ -3,7 +3,7 @@
 {
 	"name": "PHP",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/php:0-8.2",
+	"image": "mcr.microsoft.com/devcontainers/php:1-8.3",
```

Comment:
Can you fix the merge conflicts?

Reply:
merge conflict resolved, I have rebased my branch on latest main branch.